app/views/layouts/application.html.erb

```diff
@@ -15,6 +15,8 @@
         <%= msg %>
       </div>
     <% end %>
+    <img src=<%= ApplicationController.load_image.url %>>
     <%= yield %>
+
```

Comment:
<img> tells it it's gonna be an image with a source
the src is all the pieces fit together 

the controller, the method to format the data, and the pulling out the specific object we want, and place in the view for the world to see